appengine/headless-chrome/README.md

```diff
@@ -0,0 +1,12 @@
+# Headless Chrome on Google App Engine
+
+This sample application demonstrates how to use Headless Chrome via the [Puppeteer](https://developers.google.com/web/tools/puppeteer/) module to take screenshots of webpages on [Google App Engine](https://cloud.google.com/appengine) Node.js [standard environment](https://cloud.google.com/appengine/docs/standard/nodejs).
```

Comment:
I think the prerequisites are missing here. Do we not link to the `gcloud` "Before you get started" section in these samples?

Reply:
No we do not.
But I am happy to add it.

Reply:
"Refer to the appengine/README.md file for more instructions on running and deploying this app" is the sentence most examples in the repo use. Can we do the same? Then we also have the section on credentials.

Reply:
Sounds good, I just pushed a change that point to the main README